wazuh/wazuh-splunk
# Error message on agents tab

**Description:**

When connecting to the forwarder, sometimes get an error message saying  `Unable to retrieve results. It may be due to a connection problem with the Splunk forwarder, please try restarting this service.`  which does not have to appear when the connection is made.

**Pre-conditions:**

have connection with the forwarder

**Test:**

1. log in
2. Navigate to `Agents`
3. See if the error appears
4. Navigate to `Security Events`
5. See if the error appears
6. Repeat several times

**issue:**

- #1299 1299

**Screenshots**


https://user-images.githubusercontent.com/63758389/166700807-2b3240c3-9a7e-4e92-bb7e-d01dd690c59d.mp4
